### PR TITLE
feat: PL to GitHub sync

### PR DESCRIPTION
**Pull request by pawanpaudel93**
## Summary

This PR adds a feature to sync repositories from PL to Github. Also, it uses User's full name and email address if available while doing the commits.

1. Follow [https://github.com/pawanpaudel93/protocol-land-sync-github?tab=readme-ov-file#protocol-land-sync-github](https://github.com/pawanpaudel93/protocol-land-sync-github?tab=readme-ov-file#protocol-land-sync-github)
2. Also add a contributor to repo, accept the invite by the contributor and now owner should check Github Sync settings to allow the pending contributors to use GitHub Sync. Now, contributor can make new commits and the GitHub workflow trigger will occur.
3. Disable the GitHub Sync to disable the trigger of workflow during the commits. Now you can use the manual trigger button to trigger sync when needed from GitHub Sync settings.
4. Also test Remote helper PR: [https://github.com/labscommunity/protocol-land-remote-helper/pull/23](https://github.com/labscommunity/protocol-land-remote-helper/pull/23)

### CODE REVIEW

File: src/components/CommitRow/index.tsx [hunk 1]
```diff
@@ -1,21 +1,29 @@
 import { formatDistanceToNow } from 'date-fns'
 import { Link } from 'react-router-dom'
 
+import { isArweaveAddress } from '@/helpers/isInvalidInput'
 import { resolveUsernameOrShorten } from '@/helpers/resolveUsername'
 import { CommitResult } from '@/types/commit'
 
 export default function CommitRow({ commit }: { commit: CommitResult }) {
+  console.log(commit)
```
Comment (7i7o) [inline src/components/CommitRow/index.tsx:9]: Is this only for debugging purposes?

File: src/helpers/constants.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const CONTRACT_TX_ID = 'w5ZU15Y2cLzZlu3jewauIlnzbKw-OAxbN9G5TbuuiDQ'
+export const CONTRACT_TX_ID = 'B8gmo1897cyCjShP7QRe1XndatqLMieeymxehBK_oL8'
```
Comment (7i7o) [inline src/helpers/constants.ts:1]: Remember to evolve contract and change this back

File: scripts/deploy.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ const __dirname = path.resolve()
 // read contract source logic from 'handle.js' and encode it
 const state = fs.readFileSync(path.join(__dirname, 'warp/protocol-land/initial-state.json'), 'utf-8')
 const contractSource = fs.readFileSync(path.join(__dirname, 'contracts-dist/repository-contract.js'), 'utf-8')
-const contract = warp.contract('w5ZU15Y2cLzZlu3jewauIlnzbKw-OAxbN9G5TbuuiDQ').connect(key)
+const contract = warp.contract('B8gmo1897cyCjShP7QRe1XndatqLMieeymxehBK_oL8').connect(key)
```
Comment (7i7o) [inline scripts/deploy.js:23]: Remember to evolve contract and change this back

**Review by 7i7o — APPROVED** [review]
tACK 🚀

File: src/components/CommitRow/index.tsx (hunk 1 above)
Comment (pawanpaudel93) [reply src/components/CommitRow/index.tsx:9]: Yes. Will remove it.

**Review by kranthicodes — APPROVED** [review]
tACK

**Comment by kranthicodes** [comment]
please revert the contract details if no other changes to make.

**Pull request merged**